File: auditHub/src/main/java/com/thy/audithub/dto/DashboardEntry.java
package com.thy.audithub.dto;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class DashboardEntry {
    private String mudurluk          = "";
    private String yearMonth         = ""; // "2026-01"
    private int    totalTasks        = 0;
    private int    formatUygun       = 0;
    private int    formatUygunDegil  = 0;
    private int    formatHesap       = 0;
    private int    icerikUygun       = 0;
    private int    icerikUygunDegil  = 0;
    private int    icerikHesap       = 0;
    private int    btaCount          = 0;
    private String lastUpdated       = "";
}

File: Models/DashboardEntry.cs
namespace AuditHub.Models;

public class DashboardEntry
{
    public string Mudurluk { get; set; } = "";
    public string YearMonth { get; set; } = "";
    public int TotalTasks { get; set; }
    public int FormatUygun { get; set; }
    public int FormatUygunDegil { get; set; }
    public int FormatHesap { get; set; }
    public int IcerikUygun { get; set; }
    public int IcerikUygunDegil { get; set; }
    public int IcerikHesap { get; set; }
    public int BtaCount { get; set; }
    public string LastUpdated { get; set; } = "";
}
